package.json

```diff
@@ -69,10 +69,10 @@
     "sinon-chai": "2.8.0"
   },
   "dependencies": {
-    "@parity/abi": "2.1.x",
+    "@parity/abi": "^2.1.4",
```

Comment:
Really small detail (again). Change this to 2.1.x or ~2.1.4 (^2.1.4 signifies something else)

Reply:
Sure!